src/inline.d

```diff
@@ -35,8 +35,10 @@ import ddmd.tokens;
 import ddmd.visitor;
 
 private:
-enum LOG = false;
-enum CANINLINE_LOG = false;
+
+enum LOG = 0;
+enum CANINLINE_LOG = 0;
+enum EXPANDINLINE_LOG = 0;
```

Comment:
Why switch from bools to integers? I don't see any code that relies on them being anything but Boolean.